packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -992,7 +993,13 @@ export interface DocumentsMain {
 
 export interface EnvMain {
     $getEnvVariable(envVarName: string): Promise<string | undefined>;
+    $getAllEnvVariables(): Promise<EnvVariable[]>
```

Comment:
I am not sure why new APIs are necessary. Do you implement some VS Code API with them?

Reply:
Probably I changed formatting or so for this line. It was added long time ago.

Reply:
Not sure what you mean, there are no such APIs on https://github.com/eclipse-theia/theia/blob/4d4f6c293907893670c7aa7c11b55c4811d004a3/packages/plugin-ext/src/common/plugin-api-rpc.ts#L993-L996 and there is no formatting and whitespace changes

Reply:
My mistake, sorry. Will delete them...